### PR TITLE
Moving average query guide

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/query-data/flux/moving-average.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+// OR
+
+data
+  |> timedMovingAverage(every: 5m, period: 10m)
+```
+
+#### movingAverage()
+For each row in a table, `movingAverage()` returns the average of the current
+value and the previous `n-1` values.
```
Comment (kelseiv) [inline content/v2.0/query-data/flux/moving-average.md:43]: where n is the number of rows?

File: content/v2.0/query-data/flux/moving-average.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+#### timedMovingAverage()
+For each row in a table, `timedMovingAverage()` returns the the average of the
```
Comment (kelseiv) [inline content/v2.0/query-data/flux/moving-average.md:46]: double the

File: content/v2.0/query-data/flux/moving-average.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    period: 4m
+  )
+```
+{{% /flex-content %}}
+{{% flex-content %}}
+**Output:**  
+
+| _time                | _value |
+|:-----                | ------:|
+| 2020-01-01T00:04:00Z | 1.23   |
```
Comment (kelseiv) [inline content/v2.0/query-data/flux/moving-average.md:105]: Where current time is n, the timed moving average is calculated for each n - period, increasing by the specified `every` interval until all input rows have been averaged. For example:

timed moving average = 
[(value at n) + (value at each point between n- period)]/number of points (rows),
[(value at n + every) + (value at each point between n + every - period)]/number of points (rows)
...

File: content/v2.0/query-data/flux/moving-average.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+| 2020-01-01T00:05:00Z | 1.4    |
+| 2020-01-01T00:06:00Z | 2.0    |
+
+**The following functions would return:**
+
+---
+
+{{< flex >}}
+{{% flex-content %}}
+**Function:**
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/v2.0/query-data/flux/moving-average.md:67]: I restructured these examples.

File: content/v2.0/query-data/flux/moving-average.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+---
+
+{{< flex >}}
+{{% flex-content %}}
+**Function:**  
+
+```js
+// ...
+  |> timedMovingAverage(
+    every: 2m,
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/v2.0/query-data/flux/moving-average.md:95]: Minutes are correct here. The sample data increments by minutes.

File: content/v2.0/query-data/flux/moving-average.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+    period: 4m
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/v2.0/query-data/flux/moving-average.md:96]: Same

File: content/v2.0/query-data/flux/moving-average.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (kelseiv) [reply content/v2.0/query-data/flux/moving-average.md:95]: Curious how does the function determine when to calculate output? When comparing the sample input and output data, it appears the function grabs 4 hours of input data, calculates an average to output, and then advances 2 hours and does another calculation for output.

File: content/v2.0/query-data/flux/moving-average.md (hunk 3 above)
Comment (sanderson) [reply content/v2.0/query-data/flux/moving-average.md:120]: I added a visual representation instead:

image

Comment (kelseiv) [reply content/v2.0/query-data/flux/moving-average.md:120]: excellent, looks great!

File: content/v2.0/query-data/flux/moving-average.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (kelseiv) [reply content/v2.0/query-data/flux/moving-average.md]: @sanderson, sorry, I was reading the minute as an hour in the timestamp

**Pull request merged**